Applications/Spire/Source/Ui/EditableBox.cpp

```diff
@@ -171,21 +182,13 @@ void EditableBox::select_all_text() {
 }
 
 void EditableBox::on_focus(FocusObserver::State state) {
-  if(isHidden()) {
+  if(m_input_box->isHidden()) {
     return;
   }
   if(state == FocusObserver::State::NONE) {
```

Comment:
Avoid doing things like:

    if(condition) {
      action(true);
    } else {
      action(false);
    }

Instead it's just:

    action(condition);